### ISSUE TITLE
Sort the birthdays on the activeness of a member

### ISSUE THREAD

**Issue by gafderks** [GEWIS/gewisweb#787]
Currently the item in the list of birthdays corresponding to the image is not shown on top. This can be fixed by sorting the birthdays on the activeness of the members. This way also the 'most relevant' birthdays are listed first.

Related to #783

**Comment by raketwissenschaftler** [comment]
Do we already have an accurate metric for "Activeness"? I get that sometimes the picture of a inactive member is shown instead of a very active member, but what if, for example, somebody who is doing 3 committees for 3 months, has his birthday on the same day as someone who has been in the BAC for 5 years already? Who is the most active in this case?

**Comment by armandduijn** [comment]
Could you describe an algorithm to determine this? This seems like quite some work because you would need to take into account how long a member is part of a committee. For example, intuitively a BAC member for 5 years is more active than a second year in 3 different committees.

**Comment by gafderks** [comment]
There is not need to define a metric for activeness. This issue is just about putting the name of the member on the picture on top. This must be possible as we have the tag information.

**Comment by raketwissenschaftler** [comment]
Well, that is exactly the discussion I want to start: What would be a good algorithm to determine this? I agree with you that a 5 year BAC-member is more active than a second year in three committees. But what if we have someone that has been a member of the AC, AViCo and GEILER for 5 years and someone who has been a BAC-member for 5 years? Not to discredit one of the committees mentioned above, but I think that the workload of the BAC is higher than those 3 combined. 

To migate that issue, we would have to create a certain ranking of how much "activeness-points" you get for a certain committee, which, to my opinion, isn't really anyone's task, and certainly not one of the web committee. 

A solution I would propose by not showing just one picture of somebody who is celebrating his birthday, but making it random with a bias: If you are more active, which now can be a pretty loose metric, your picture will be shown more often.

**Comment by armandduijn** [comment]
@raketwissenschaftler I agree that ranking committees shouldn't be the responsibility of the WC. I agree with your proposed solution but determining who is "more active" is again non-trivial. Instead, I propose we just randomize the birthday picture. The only bias is if the member is active or not.

**Comment by raketwissenschaftler** [comment]
I am fine with either of them, only with the randomized solution I would still add the non-active members, but add a bias to active members. For example, it picks an active member 70% of the time and a non-active member 30% of the time.

Although, typing this I have concluded that that is just unnecessarily complicated and therefore I think that just randomizing it is a good solution.

The way you explain it, @gafderks has a good solution as well, the only drawback is that in the future(say, 2025), we will have lots of older members that still have a lot of tags, but are already driving around in a stationwagon with a golden retriever in the back, so their pictures will not be as interesting for the current members.
